src/scenic/simulators/carla/model.scenic

```diff
@@ -143,3 +154,146 @@ class Trash(Prop):
 
 class Cone(Prop):
     blueprint: Uniform(*blueprints.coneModels)
+
+
+class Debris(Prop):
+    blueprint: Uniform(*blueprints.debrisModels)
+    physics: False
+
+
+class VendingMachine(Prop):
+    blueprint: Uniform(*blueprints.vendingMachineModels)
+
+
+class Chair(Prop):
+    blueprint: Uniform(*blueprints.chairModels)
+
+
+class BusStop(Prop):
+    blueprint: Uniform(*blueprints.busStopsModels)
+
+
+class Advertisement(Prop):
+    blueprint: Uniform(*blueprints.advertisementModels)
+
+
+class Garbage(Prop):
+    blueprint: Uniform(*blueprints.garbageModels)
+
+
+class Container(Prop):
+    blueprint: Uniform(*blueprints.containerModels)
+
+
+class Table(Prop):
+    blueprint: Uniform(*blueprints.tableModels)
+
+
+class Barrier(Prop):
+    blueprint: Uniform(*blueprints.barrierModels)
+
+
+class PlantPot(Prop):
+    blueprint: Uniform(*blueprints.plantpotModels)
+
+
+class Mailbox(Prop):
+    blueprint: Uniform(*blueprints.mailboxModels)
+
+
+class Gnome(Prop):
+    blueprint: Uniform(*blueprints.gnomeModels)
+
+
+class CreasedBox(Prop):
+    blueprint: Uniform(*blueprints.creasedboxModels)
+    physics: False
+
+
+class Case(Prop):
+    blueprint: Uniform(*blueprints.caseModels)
+
+
+class Box(Prop):
+    blueprint: Uniform(*blueprints.boxModels)
+
+
+class Bench(Prop):
+    blueprint: Uniform(*blueprints.benchModels)
+
+
+class Barrel(Prop):
+    blueprint: Uniform(*blueprints.barrelModels)
+
+
+class ATM(Prop):
+    blueprint: Uniform(*blueprints.atmModels)
+
+
+class Kiosk(Prop):
+    blueprint: Uniform(*blueprints.kioskModels)
+
+
+class IronPlate(Prop):
+    blueprint: Uniform(*blueprints.ironplateModels)
+    physics: False
+
+
+class TrafficWarning(Prop):
+    blueprint: Uniform(*blueprints.trafficwarningModels)
+
+
+## Utility functions
+
+def freezeTrafficLights():
+    """ Freezes all traffic lights in the scene. Frozen traffic lights can be modified by the user
+    but the time will not update them until unfrozen. """
+    simulation().world.freeze_all_traffic_lights(True)
+
+def unfreezeTrafficLights():
+    """ Unfreezes all traffic lights in the scene. """
+    simulation().world.freeze_all_traffic_lights(False)
+
+def _getClosestLandmark(vehicle, type, distance=100):
+    if vehicle._intersection is not None:
+        return None
+
+    waypoint = simulation().map.get_waypoint(vehicle.carlaActor.get_transform().location)
+    landmarks = waypoint.get_landmarks_of_type(distance, type)
+
+    if landmarks:
+        return builtins.min(landmarks, key=lambda l: l.distance)
```

Comment:
This is a flat bug in Scenic - you should be able to just write `min`, but I forgot that it accepts keyword arguments. I've pushed a fix.